extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -497,4 +519,143 @@ private boolean isEqual(State state1, State state2) {
         return colorModeIsEqual && effectIsEqual;
     }
 
+	@Override
+	public Collection<ConfigStatusMessage> getConfigStatus() {
+		logger.debug("IP configuration invoked.");
+		
+		// Error handler bound to the IP address field
+		ConfigStatusInfo configStatus = new ConfigStatusInfo();
+		ConfigStatusMessage.Builder errBuilder = ConfigStatusMessage.Builder.error(HueBindingConstants.HOST);
+		
+		// Bridge properties' map and IP address to be used further throughout checks
+		Map<String, String> bridgeProperties = editProperties();
+		final String bridgeIpAddress = (String) getThing().getConfiguration().get(HueBindingConstants.HOST);
+		
+		// Check whether an IP address is provided
+        if (bridgeIpAddress == null) {
+            logger.error("No IP address has been provided.");
+            configStatus.add(errBuilder.withMessageKey(HueErrorCodeDescriptor.CONFIG_FLOW_ERROR_PREFIX 
+            		+ HueErrorCodeDescriptor.IP_ADDRESS_MISSING.getCode()).build());
+	    	
+            return configStatus.getConfigStatusMessages();
+        }
+
+        // Check if the bridge on the given IP address is reachable
+        if (! isReachable(bridgeIpAddress, DEFAULT_BRIDGE_REACH_TIMEOUT)) {
+            logger.error("The bridge is not responding and is not reachable on {}.", bridgeIpAddress);
+        	configStatus.add(errBuilder.withMessageKey(HueErrorCodeDescriptor.CONFIG_FLOW_ERROR_PREFIX 
+        			+ HueErrorCodeDescriptor.BRIDGE_NOT_RESPONDING.getCode()).build());
+	    	
+            return configStatus.getConfigStatusMessages();
+        }
+
+        // The hue bridge is reachable. So let's obtain its serial number
```

Comment:
we should delete the following code of the getConfigStatus operation because of
- the serial number is already during the initialization... we dont expect any configuration issue here
- we dont expect that the thing is null here (```Bridge bridge = getThing()```)
- we dont want to edit or update the properties during config status determination